card-admin/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="payment">
+            <group id="pronko_liqpay_section" sortOrder="6" showInDefault="1" showInWebsite="1" showInStore="1">
+                <group id="pronko_liqpay" translate="label" type="text" sortOrder="100" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>LiqPay</label>
+                    <fieldset_css>complex</fieldset_css>
+                    <attribute type="displayIn">other_payment_methods</attribute>
```

Comment:
attribute node can be removed.

Reply:
After discussion we decide to leave them for now.